build/js/intlTelInput.js

```diff
@@ -216,6 +218,15 @@ https://github.com/Bluefieldscom/intl-tel-input.git
                 this.countryList.removeClass("v-hide").addClass("hide");
                 // this is useful in lots of places
                 this.countryListItems = this.countryList.children(".country");
+                // create dropdownContainer markup
+                if (this.options.dropdownContainer) {
+                    this.$drop = $("<div>", {
```

Comment:
We don't use $ variable names anywhere else - please remove the $. Also let's call this variable "dropdown" instead of just "drop".